packages/wonder-blocks-dropdown/components/action-menu-opener.js

```diff
@@ -16,7 +16,7 @@ export type SharedProps = {|
     /**
      * Whether the opener is disabled. If disabled, disallows interaction.
      */
-    disabled?: boolean,
+    disabled: boolean,
```

Comment:
`"react/default-props-match-prop-types": ["error", { "allowRequiredDefaults": true }]` found this, but I ended up not including this rule b/c there was a false positive.